src/Components/Components/src/Reflection/ComponentProperties.cs

```diff
@@ -168,11 +168,14 @@ internal static IEnumerable<PropertyInfo> GetCandidateBindableProperties([Dynami
         var propertyInfo = targetType.GetProperty(parameterName, BindablePropertyFlags);
         if (propertyInfo != null)
         {
-            if (!propertyInfo.IsDefined(typeof(ParameterAttribute)) && !propertyInfo.IsDefined(typeof(CascadingParameterAttribute)))
+            if (!propertyInfo.IsDefined(typeof(ParameterAttribute)) &&
+                !propertyInfo.IsDefined(typeof(CascadingParameterAttribute)) &&
+                !propertyInfo.IsDefined(typeof(SupplyParameterFromFormAttribute)))
```

Comment:
Merging HTML form concepts directly into the core renderer at such a low level isn't ideal. Especially not including info about it in error messages, when this is meant to work for technologies like WebView where form posts aren't a concept.

Can we look at having something like making `CascadingParameterAttribute` nonsealed so that `SupplyParameterFromFormAttribute` can live in M.A.C.Web and derive from it, and the core renderer wouldn't have to know? Or if necessary have an interface that attributes can implement to signal that they are about consuming a cascading parameter.

Reply:
Would it work if we had an internal marker interface for this instead? I don't want to unseal `CascadingValue` and start allowing people to inherit from it.

I think that we can potentially put this in `Microsoft.AspNetCore.Components.Web`, but I would also say that I don't know how worth doing so is. There is going to be a limited number of these, and if we want or need to in the future, we can always move, and type forward them?

Reply:
The reason I am asking is because that adds a non trivial amount of complexity to the implementation and I want to avoid doing so at the moment.

Reply:
> I don't want to unseal CascadingValue and start allowing people to inherit from it.

I wasn't proposing that - I was proposing unsealing `CascadingParameterAttribute`, since here we're concerned with making the renderer understand that other things like `[FormParameter]` should also receive data from cascaded values. It shouldn't be hugely complex, unless I'm missing something major.